index.html

```diff
@@ -25,11 +25,11 @@
     <main>
         
         <section class="homepage">
-        <h1><span class="part1-main-title">Hi! I'm <span class="emphasize-rosa">Rosa,</span></span><br>a full stack developer<br>based in Bristol.</h1>
+            <h1><span class="part1-main-title">Hi! I'm <span class="emphasize-rosa">Rosa,</span></span><br>a <span class="soon-to-be">(soon-to-be)</span> full stack developer<br>based in Bristol.</h1>
```

Comment:
Using breaks is naughty and I imagine Rayna will pull you up on it. I can see why you did it. Looks legit

Reply:
You could set a width on the parent section perhaps?